app/assets/stylesheets/application.scss

```diff
@@ -44,6 +57,7 @@ ol {
 }
 
 footer {
+  font-size: 0.75em;
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets<br>Properties should be ordered background-color, color, font-size, opacity, padding